test-api/schema-basic/User.js

```diff
@@ -275,6 +275,9 @@ const User = new GraphQLObjectType({
     },
     writtenMaterial1: {
       type: new GraphQLList(AuthoredUnion),
+      args: {
+        search: { type: GraphQLString },
```

Comment:
Excellent to add to the tests ... can you explain what is being added to test this?

Reply:
This was done to be able to test multiple aliases with different args on union and interface types. I added a comment explaining this.